manifests/mroute.pp

```diff
@@ -77,6 +87,13 @@
     default =>  $route_down_template,
   }
 
+  if $::osfamily == 'SuSE' {
+    $networks = keys($routes)
+    network::mroute::validate_gw { $networks:
```

Comment:
Why a define just to validate if the route is a valid string?

Reply:
This is done due to tests which run under puppet 3.x.
There is a special comment in `validate_gw` define:

> We need it to have compatibility with puppet 3.x, 'cause it doesn't support each.